lib/robots/dor_repo/speech_to_text/stt_create.rb

```diff
@@ -27,8 +27,7 @@ def perform_work
         def send_sqs_message
           message_body = {
             id: job_id,
-            druid:,
-            media:
+            druid:
```

Comment:
leaving the druid in the job message (even though it happens to be included in the job_id now) ... maybe useful for logging or other purposes to have it distinct